projects/ngrx.io/content/guide/entity/recipes/additional-state-properties.md

```diff
@@ -0,0 +1,84 @@
+# Additional Entity State Properties Update
+
+For beginners it is very confusing how to update the additional state properties using adapter. So we are going to see how to update the Additional Entity state properties. In the below example we are going to use the [Entity Adapter](https://ngrx.io/guide/entity/adapter) example.
+
+Usage:
+
+we are declaring the `selectedUserId` as an additional entities state property.
+
+<code-example header="user.reducer.ts">
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';

```

Comment:
Could you update these examples in the "NgRx 8" way please.
For reference, you can take a look at https://github.com/ngrx/platform/pull/1856 or ask help here.

Reply:
updated to NgRx 8